tests/integration/security/testdata/requestauthn/f-authn.yaml.tmpl

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
+metadata:
+  name: "request-authn-for-f"
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: f
+  jwtRules:
+  - issuer: "test-issuer-1@istio.io"
+    jwks: '{ "keys":[ {"e":"AQAB","kid":"abc","kty":"RSA","n":"failedToFetchJwtPubKeyhttpsrawgithubusercontentcomistioistiorelease16securitytoolsjwtsamplesjwksjson"}]}'
```

Comment:
Can you update to use an non-exist jwksUri instead of using jwks?

Reply:
working on it. Thanks!